### PR TITLE
Fixed #14402 -- Fixed not displaying help_text for ManyToMany raw_id_fields.

### PR DESCRIPTION
**Pull request by sasha0**

### CODE REVIEW

**Comment by timgraham** [comment]
Did you investigate why the removed code was added in the first place?
It'd be better to test this in a non-selenium test as this functionality doesn't require it and the selenium tests aren't run in the normal build.

**Comment by sasha0** [comment]
> Did you investigate why the removed code was added in the first place?

I did not find any explanation, `help_text` was set to blank from the beginning - 
https://github.com/django/django/blob/a19ed8aea395e8e07164ff7d85bd7dff2f24edca/django/contrib/admin/options.py#L167

> It'd be better to test this in a non-selenium test as this functionality doesn't require it and the selenium tests aren't run in the normal build.

Yes, but instead of adding new test case for this model, I thought it'd cleaner just to add few lines to the existing test, which checks `raw_id_fields` functionality for M2M, which appeared to be selenium test.

File: tests/admin_widgets/models.py [hunk 1]
```diff
@@ -68,7 +68,8 @@ class Event(models.Model):
         limit_choices_to=models.Q(pk__gt=0),
         related_name='events_main_band_at',
     )
-    supporting_bands = models.ManyToManyField(Band, blank=True, related_name='events_supporting_band_at')
+    supporting_bands = models.ManyToManyField(Band, blank=True, related_name='events_supporting_band_at',
```
Comment (timgraham) [inline tests/admin_widgets/models.py]: This should use hanging indent style:
```python
supporting_bands = models.ManyToManyField(
    Band,
    related_name='events_supporting_band_at',
    ...,
)
```

**Comment by timgraham** [comment]
I guess the test is fine...  this doesn't seem likely to regress anyway.

File: tests/admin_widgets/models.py (hunk 1 above)
Comment (sasha0) [inline tests/admin_widgets/models.py]: Fixed.

File: tests/admin_widgets/tests.py [hunk 2]
```diff
@@ -1255,6 +1255,12 @@ def test_many_to_many(self):
             self.selenium.find_element_by_id('id_supporting_bands').get_attribute('value'),
             '')
 
+        # Help text for the field is displayed
+        self.assertEqual(
+            self.selenium.find_element_by_css_selector('.field-supporting_bands span.help').text,
```
Comment (timgraham) [inline tests/admin_widgets/tests.py]: Looks like this should be `p.help` not `span`.

File: tests/admin_widgets/models.py [hunk 3]
```diff
@@ -68,7 +68,11 @@ class Event(models.Model):
         limit_choices_to=models.Q(pk__gt=0),
         related_name='events_main_band_at',
     )
-    supporting_bands = models.ManyToManyField(Band, blank=True, related_name='events_supporting_band_at')
+    supporting_bands = models.ManyToManyField(
+        Band,
+        blank=True,
+        related_name='events_supporting_band_at',
+        help_text='Supporting Bands.')
```
Comment (timgraham) [inline tests/admin_widgets/models.py]:
```
   help_text='Supporting Bands.',
)
```

**Comment by timgraham** [comment]
merged in c79852acee9a2ca137ff0e19657342c7406bbbb9, thanks!

**Pull request closed without merging**